### PR TITLE
sync only when unmetered internet is available? #1670

### PR DESCRIPTION
**Pull request by advaita13**
Created a Sync Only on Wi-Fi switch in Settings. When On, a receiver is
created when the sync is pending and Wi-Fi is disconnected. This
listens for any Wi-Fi state change broadcasts. Once connected the
receiver is disabled.

Reference : #1670

### CODE REVIEW

**Comment by advaita13** [comment]
Reference : https://github.com/open-keychain/open-keychain/issues/1670

File: OpenKeychain/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -5,6 +5,11 @@
     android:installLocation="auto">
 
     <!--
+        General remarks
+        ===============
+        - Last APG 1 version was 10900 (1.0.9 beta 00)
+        - Keychain starting with versionCode 20000!
+
```
Comment (dschuermann) [inline OpenKeychain/src/main/AndroidManifest.xml]: These have been removed on git master, you somehow rebased your commits in the wrong way.

File: OpenKeychain/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1733,7 +1734,6 @@
     <string name="help_donation_paypal_item">OpenKeychain Donation</string>
     <string-array name="help_donation_google_catalog_values">
         <item>1 EUR</item>
-        <item>2 EUR</item>
```
Comment (dschuermann) [inline OpenKeychain/src/main/res/values/strings.xml]: Same as my other comment. This has just been added on git master

**Comment by advaita13** [comment]
On account of deletion of a few lines and rearrangement of import statements the commits are showing unnecessary deletions from the Git master. I'll close this request and open up another one with proper changes. Apologies.

**Pull request closed without merging**